### DISCUSSION TITLE
multicast IP(s)

### DISCUSSION THREAD

**Discussion by sbj-ee** [netbox-community/netbox#10906] [Q&A]
I manage video platforms, so we have lots of multicast IPs in use.

I was curious how others are adding them into netbox.

I’ve been adding them all as a /32 and this seems to work nicely.

Anybody doing this differently which works better?

**Comment by candlerb** [comment]
That's the reasonable approach.  You can make a fake aggregate and/or prefix for 224.0.0.0/4 if you want.  But the individual addresses are /32's.
